src/components/TodosList.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react';
+import { CompleteTodo } from '../constants/types';
+import './TodoList.css';
+
+interface Props {
+  todos: CompleteTodo[];
+}
+
+export const TodosList: FC<Props> = (props) => {
+  const { todos } = props;
+
+  return (
+    <ul>
+      {todos.map((todo: CompleteTodo) => (
```

Comment:
если указать корректные пропсы, это будет не нужно
```suggestion
      {todos.map((todo) => (
```

Reply:
Исправил